src/unicef_locations/models.py

```diff
@@ -82,10 +88,12 @@ class Location(MPTTModel):
         blank=True,
     )
     point = models.PointField(verbose_name=_("Point"), null=True, blank=True)
+    is_active = models.BooleanField(verbose_name=_("Active"), default=True, blank=True)
     created = AutoCreatedField(_('created'))
     modified = AutoLastModifiedField(_('modified'))
 
-    objects = LocationManager()
+    objects = ActiveLocationsManager()
+    all_locations = AllLocationsManager()
```

Comment:
why do we need two managers?

Reply:
Because the `objects` manager fetches only the active locations, and in few places(Django admin,  location import task, maybe location GIS endpoints in the `management` app) we need to have archived locations too in the queryset.

Reply:
check InterventionManager in eTools, you can define a custom manager and define custom filters

Reply:
Ok, i am removing the extra manager then and make it similar to InterventionManager.